test/functional/feature_assumeutxo.py

```diff
@@ -130,8 +130,8 @@ def no_sync():
 
         monitor = n1.getchainstates()
         assert_equal(monitor['normal']['blocks'], START_HEIGHT)
-        assert_equal(monitor['snapshot']['blocks'], SNAPSHOT_BASE_HEIGHT)
-        assert_equal(monitor['snapshot']['snapshot_blockhash'], dump_output['base_hash'])
+        assert_equal(monitor.get('snapshot', {}).get('blocks'), SNAPSHOT_BASE_HEIGHT)
+        assert_equal(monitor.get('snapshot', {}).get('snapshot_blockhash'), dump_output['base_hash'])
```

Comment:
`snapshot` is an optional key in the RPC's return value, so if the above call returns before a snapshot chainstate is loaded, it will `KeyError` as I've shown in https://github.com/bitcoin/bitcoin/issues/28585.

Reply:
Oh I see what you're saying - the test will still fail; of course... Reverting those.